### PR TITLE
fix : cards even and resources page fixed

### LINKED ISSUE #352: Bug: Cards are not event
image

Uneven cards

### PR DESCRIPTION
**Pull request by sunidhisachan**
image
Closes #352

### CODE REVIEW

**Review by redxzeta — CHANGES_REQUESTED** [review]
fix your pr title to `fix: card even`

image

Looks too long. maybe it would be better to dl the images and crop them to all have the same width and height and then add them to the `charities.json`

**Comment by redxzeta** [comment]
@sunidhisachan are u still going to work on this?

**Comment by sunidhisachan** [comment]
> @sunidhisachan are u still going to work on this?

Yes , I'll do it soon

**Comment by sunidhisachan** [comment]
image
  
@redxzeta  I have made the changes but unable to push them .

File: src/App.css [hunk 1]
```diff
@@ -30,3 +30,12 @@
     transform: none;
   }
 }
+.card-container {
+  height: 325px;
+}
+.title-container {
+  height: 100px;
+}
+.btn{
```
Comment (redxzeta) [inline src/App.css:39]: You could add `m-2.5` to the className

File: src/App.css [hunk 2]
```diff
@@ -30,3 +30,12 @@
     transform: none;
   }
 }
+.card-container {
+  height: 325px;
+}
+.title-container {
+  height: 100px;
```
Comment (redxzeta) [inline src/App.css:37]: tw has some classes for height `h-24` , `h-80` 

https://tailwindcss.com/docs/height

useful tailwind vscode extension: https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss

**Review by redxzeta — CHANGES_REQUESTED** [review]

**Pull request closed without merging**